### PR TITLE
Some features

### PR DESCRIPTION
**Pull request by wffranco**
* Module::isActive('module') let you validate if a module exists and is active but without generating error.

* Routes folder, configured as in Laravel 5.3+ (and according to the standard, the routes should be loaded in the service provider, so the start.php file is still present but empty).

* No Namespace folders (like config & routes) must be in lowercase.

### CODE REVIEW

File: src/Repository.php [hunk 1]
```diff
@@ -553,6 +553,18 @@ public function notActive($name)
...
     /**
+     * Determine if the given module exist and is activated.
+     *
+     * @param string $name
+     *
+     * @return bool
+     */
+    public function isActive($name)
+    {
+        return ($module = $this->find($name)) && $module->active();
```
Comment (nWidart) [inline src/Repository.php]: The `active` method does this already.

**Comment by nWidart** [comment]
Hello and thanks for the contribution.

However please consider sending one PR per feature/change you make.

File: src/Repository.php (hunk 1 above)
Comment (wffranco) [inline src/Repository.php]: Yes but the actual "active" method throw an error if the module is not present.

The new method let you validate if the module exist and is active.

Example: I need to create a menu based on 3 optional modules. So if a partner install just 2 modules, the menu throw an error with the old active method. The new one just return true or false.

**Comment by wffranco** [comment]
Except for the "isActive" method, all the other files are related, and should be in the same pull request.

File: src/Repository.php (hunk 1 above)
Comment (wffranco) [inline src/Repository.php]: I preserve the original functionality in case somebody else need it that way.

Comment (nWidart) [inline src/Repository.php]: It will throw a `ModuleNotFoundException` exception, which you can catch to build your menu.

**Comment by nWidart** [comment]
Sorry but after further review I can't accept this PR. Routes are loaded in a RouterServiceProvider, which can be generated with this package. This is the usual recommended way instead of via `start.php`.

This also gives the package consumer the flexibility of having the routes file at any desired location.
This is the stub file for the routeSP https://github.com/nWidart/laravel-modules/blob/master/src/Commands/stubs/route-provider.stub
Which is called by the `module:route-provider` command.

File: src/Repository.php (hunk 1 above)
Comment (wffranco) [inline src/Repository.php]: I understand your point, but the main purpose of laravel is easy coding.
So if you need a fast validation, this:
```php
if (Module::isActive('module') {
    // do something
}
```
is faster that:
```php
try {
    if (Module::active('module') {
        // do something
    }
} catch (ModuleNotFoundException $e) {
}
```
that for every module I use.

That behavior make me add this extra code in every module for exceptions that I don't need to catch, I just want to ignore some code if the module don't exist or is disabled.

**Comment by wffranco** [comment]
When I create a new module this don't have the route provider... Is a new feature or somebody disable it?

**Pull request closed without merging**